Example/StripeConnectExample/StripeConnectExample/MainViewController.swift

```diff
@@ -76,9 +77,16 @@ class MainViewController: UITableViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        title = merchant.displayName ?? merchant.merchantId
-        navigationItem.titleView = navbarTitleButton
+        title = merchant.displayName.map { "Demo account: \($0)" } ?? merchant.merchantId
```

Comment:
When the navbar just said "Custom" it wasn't clear from the context that this was a demo account, so I thought adding this would help